src/igvfd/schemas/reference_file.json

```diff
@@ -129,6 +129,22 @@
         "schema_version": {
             "default": "16"
         },
+        "catalog_adapters": {
+            "title": "IGVF Catalog Adapters",
+            "description": "IGVF Catalog Adapters that ingests this file",
+            "type": "array",
+            "permission": "admin_only",
+            "uniqueItems": true,
+            "items": {
+                "title": "IGVF Catalog Adapter Github Link",
```

Comment:
I think the title should be singular of the above title. so "IGVF Catalog Adapter" would be more analogous

Reply:
Or we could change both to say "Catalog Adapter Links" and "Catalog Adapter Link" respectively